lesson3.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('js-list');
+for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    const img = document.createElement('img');
+    const imgbox = ['/img/bookmark.png','/img/message.png'];
+    img.src = imgbox[i-1];
+    const fragment = document.createDocumentFragment();
```

Comment:
[want].
To create a new DocumentFragment
You only need to define a const fragment once.
Let's handle it outside the loop here.

Reply:
I understand now! Thank you for your comment.